src/app/api/v1/location-autocomplete/route.ts

```diff
@@ -0,0 +1,119 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { NextResponse, NextRequest } from "next/server";
+import uFuzzy from "@leeoniya/ufuzzy";
+
+import { IRelevantLocation } from "../../../../scripts/build/createLocationAutocompleteData/types";
+// The location autocomplete data will be created during the build step.
+// @ts-ignore-next-line
+// eslint-disable-next-line import/no-unresolved
+import locationData from "../../../../../locationAutocompleteData.json";
+
+export type TMatchingLocations = Array<IRelevantLocation> | [];
```

Comment:
What's `TMatchingLocations`'s `T` prefix, does that stand for `Type`?
Seems like we don't regularly use that pattern for types:

<details>
<summary><kbd>git grep -Poh "(?:export )?type \w+" -- '*.ts' '*.tsx' | sort | uniq</kbd></summary>

```sh
export type BreachResolutionTypes
export type ExtendedReactLocalization
export type FeatureFlag
export type FeatureFlagPutRequest
export type GetFragment
export type HibpBreachDataTypes
export type LocaleData
export type Props
export type TAlternateNameData
export type TLocationData
export type TMatchingLocations
type AppPickerItemProps
type AppPickerMenuProps
type AppPickerPopoverProps
type AppPickerTriggerProps
type BreachAutoInsertedColumns
type BreachOptionalColumns
type DetailsFoundProps
type EmailAddressAutoInsertedColumns
type FeatureFlagAutoInsertedColumns
type FeatureFlagOptionalColumns
type FxaProfile
type FxaSubscriptionResponse
type LogFunction
type Options
type Story
type SubscriberAutoInsertedColumns
type SubscriberOptionalColumns
type TAlternateName
type TAlternateNameId
type TFrom
type TGeonameId
type TIsAbbreviation
type TIsColloquial
type TIsHistoric
type TIsPreferredName
type TIsShortName
type TIsolanguage
type TLocationAdmin1Code
type TLocationAdmin2Code
type TLocationAdmin3Code
type TLocationAdmin4Code
type TLocationAlternatenames
type TLocationAsciiname
type TLocationCc2
type TLocationCountryCode
type TLocationDem
type TLocationElevation
type TLocationFeatureClass
type TLocationFeatureCode
type TLocationLatitude
type TLocationLongitude
type TLocationModificationDate
type TLocationName
type TLocationPopulation
type TLocationTimezone
type TTo
type assertion
type checking
```
</details>

In fact, if we filter the results to T* prefixes it looks like we only use that pattern in 1-2 files added here:

<details>
<summary><kbd>git grep -Pon "(?:export )?type \w+" -- '*.ts' '*.tsx' | sort | uniq | grep "\btype\sT"</kbd></summary>

```sh
src/app/api/v1/location-autocomplete/route.ts:14:export type TMatchingLocations
src/scripts/build/createLocationAutocompleteData/types.ts:104:type TLocationFeatureCode
src/scripts/build/createLocationAutocompleteData/types.ts:17:type TAlternateNameId
src/scripts/build/createLocationAutocompleteData/types.ts:19:type TGeonameId
src/scripts/build/createLocationAutocompleteData/types.ts:36:type TIsolanguage
src/scripts/build/createLocationAutocompleteData/types.ts:38:type TAlternateName
src/scripts/build/createLocationAutocompleteData/types.ts:40:type TIsAbbreviation
src/scripts/build/createLocationAutocompleteData/types.ts:42:type TIsPreferredName
src/scripts/build/createLocationAutocompleteData/types.ts:44:type TIsShortName
src/scripts/build/createLocationAutocompleteData/types.ts:49:type TIsColloquial
src/scripts/build/createLocationAutocompleteData/types.ts:54:type TIsHistoric
src/scripts/build/createLocationAutocompleteData/types.ts:56:type TFrom
src/scripts/build/createLocationAutocompleteData/types.ts:58:type TTo
src/scripts/build/createLocationAutocompleteData/types.ts:60:export type TAlternateNameData
src/scripts/build/createLocationAutocompleteData/types.ts:74:type TLocationName
src/scripts/build/createLocationAutocompleteData/types.ts:76:type TLocationAsciiname
src/scripts/build/createLocationAutocompleteData/types.ts:787:type TLocationCountryCode
src/scripts/build/createLocationAutocompleteData/types.ts:792:type TLocationCc2
src/scripts/build/createLocationAutocompleteData/types.ts:797:type TLocationAdmin1Code
src/scripts/build/createLocationAutocompleteData/types.ts:803:type TLocationAdmin2Code
src/scripts/build/createLocationAutocompleteData/types.ts:805:type TLocationAdmin3Code
src/scripts/build/createLocationAutocompleteData/types.ts:807:type TLocationAdmin4Code
src/scripts/build/createLocationAutocompleteData/types.ts:809:type TLocationPopulation
src/scripts/build/createLocationAutocompleteData/types.ts:811:type TLocationElevation
src/scripts/build/createLocationAutocompleteData/types.ts:818:type TLocationDem
src/scripts/build/createLocationAutocompleteData/types.ts:81:type TLocationAlternatenames
src/scripts/build/createLocationAutocompleteData/types.ts:820:type TLocationTimezone
src/scripts/build/createLocationAutocompleteData/types.ts:822:type TLocationModificationDate
src/scripts/build/createLocationAutocompleteData/types.ts:824:export type TLocationData
src/scripts/build/createLocationAutocompleteData/types.ts:83:type TLocationLatitude
src/scripts/build/createLocationAutocompleteData/types.ts:85:type TLocationLongitude
src/scripts/build/createLocationAutocompleteData/types.ts:90:type TLocationFeatureClass
```
</details>

Reply:
Yes, that’s supposed to stand for `Type`. Thanks for the callout — that is a pattern I didn’t explicitly plan on introducing. Removed with [e652217](https://github.com/mozilla/blurts-server/pull/3186/commits/e652217a7d9eca0e6b324c2fdca92e6d7dddf49a).